pom.xml

```diff
@@ -65,41 +65,45 @@
         <maven.compiler.release>8</maven.compiler.release>
 
         <!-- Legend dependency versions -->
-        <legend.engine.version>2.56.0</legend.engine.version>
+        <legend.engine.version>2.55.1-SNAPSHOT</legend.engine.version>
```

Comment:
This PR will need to be merged after the Dropwizard 2 upgrades for legend-shared and legend-engine are released.